ncm-network/src/main/pan/components/network/types/network/interface.pan

```diff
@@ -34,13 +34,25 @@ type network_interface_alias = {
     Describes the bonding options for configuring channel bonding on EL5 and similar.
 }
 type network_bonding_options = {
-    "mode" : long(0..6)
-    "miimon" : long
-    "updelay" ? long
-    "downdelay" ? long
-    "primary" ? valid_interface
-    "lacp_rate" ? long(0..1)
-    "xmit_hash_policy" ? choice('0', '1', '2', 'layer2', 'layer2+3', 'layer3+4')
+    "ad_select"         ? choice('bandwidth', 'count', 'stable')
+    "all_slaves_active" ? long(0..1)
+    "arp_interval"      ? long
+    "arp_ip_target"     ? type_ip[]
+    "arp_validate"      ? choice('none', 'active', 'backup', 'all')
+    "downdelay"         ? long
+    "fail_over_mac"     ? choice('active', 'follow', 'none')
+    "lacp_rate"         ? long(0..1)
+    "min_links"         ? long
+    "miimon"            : long
+    "mode"              : long(0..6)
```

Comment:
I can't find this on https://docs.rs/nmstate/latest/nmstate/struct.BondOptions.html#fields - is there modern doc? Or should be it switched to optional?

Reply:
that comes under bonding mode https://docs.rs/nmstate/latest/nmstate/enum.BondMode.html, 
https://docs.rs/nmstate/latest/nmstate/struct.BondConfig.html#structfield.mode

and in this example get set in link-aggregation  https://nmstate.io/examples.html#interfaces-bond, 
which is how this is being set in nmstate.pm code.